pluginExamples/EnlightenPlugin.py

```diff
@@ -88,9 +88,19 @@ def plot(self, y, x=None, title=None, color=None):
         @param title plot title, shown in legend
         @param color color of plot line
         """
+        if y is None:
```

Comment:
Please move this check to the caller.
I'm planning to support `plot(title="something", color="orange")` as a way to add arbitrary entries to the legend.

It's useful in LocalBaseline where I want to have n lines, but only one legend entry per color:
```
self.plot(title='X', color='red')
for i in range(n):
    self.plot(x=[...], y=[..])
```

Reply:
fixed